dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -26,6 +28,28 @@ const CMD_OMDB: &str = env!("CARGO_BIN_EXE_omdb");
 type ControlPlaneTestContext =
     nexus_test_utils::ControlPlaneTestContext<omicron_nexus::Server>;
 
+/// The `oximeter` list-producers command output is not easy to compare as a
+/// string directly because the timing of registrations with both our test
+/// producer and the one nexus registers. But, let's find our test producer
+/// in the list.
+fn assert_oximeter_list_producers_output(
+    output: &str,
+    ox_url: &str,
+    test_producer: IpAddr,
+) {
+    assert!(
+        output.contains(format!("Collector ID: {}", OXIMETER_UUID).as_str())
+    );
+    assert!(output.contains(ox_url));
+
+    let found = output.lines().any(|line| {
+        line.contains(PRODUCER_UUID)
+            && line.contains(&test_producer.to_string())
+    });
+
+    assert!(found, "test producer {} and producer UUID {} not found in oximeter list-producers output", test_producer, PRODUCER_UUID);
```

Comment:
Style nit: I think we need to wrap this line manually. `rustfmt` won't touch macro contents, IIRC, and this is a bit long.

Reply:
yeah, figured.